release.md

```diff
@@ -0,0 +1,61 @@
+# Release Processes
+
+As per the [Versioning and release process of the libs artifacts](https://github.com/falcosecurity/libs/blob/master/proposals/20210524-versioning-and-release-of-the-libs-artifacts.md), this repository includes different groups of artifacts that needs to be versioned and released. Moreover, those artifacts follows different versioning schemas (see the [Driver SemVer](https://github.com/falcosecurity/libs/blob/master/proposals/20210818-driver-semver.md) and [Versioning schema](https://github.com/falcosecurity/libs/blob/master/proposals/20220203-versioning-schema-amendment.md)) proposals).
+
+We have two separate release processes which occur **independently** of each other: 
+- The **drivers release**
+- The **libs release** (ie. _libsinp_ and _libscap_)
+
+The drivers release process is mainly automated and happens outside this repository. The actual building and distribution system is implemented in our [test-infra](https://github.com/falcosecurity/test-infra) by the [Driverkit Build Grid](https://github.com/falcosecurity/test-infra/tree/master/driverkit), and drivers are published to https://download.falco.org/?prefix=driver/. The [drivers versioning](#Drivers-versioning) process happens in this repo. 
```

Comment:
Kind of a nitpick, the last line confused me a bit because there's talk of the release process in a different repo, but versioning happens in this one, maybe we can change the wording a bit? The proposed change is entirely tentative and might not reflect the process correctly, so excuse me if I'm wrong on anything.
```suggestion
The [drivers versioning](#Drivers-versioning) process happens in this repository, but their release process is mainly automated and managed in our [test-infra](https://github.com/falcosecurity/test-infra). Building is implemented there as per the [Driverkit Build Grid](https://github.com/falcosecurity/test-infra/tree/master/driverkit), and drivers are published to https://download.falco.org/?prefix=driver/.
```

Reply:
Your tentative is OK!